src/py-opentimelineio/opentimelineio/media_linker.py

```diff
@@ -148,6 +149,29 @@ def link_media_reference(self, in_clip, media_linker_argument_map=None):
             media_linker_argument_map=media_linker_argument_map
         )
 
+    def is_default_linker(self):
+        return os.environ.get("OTIO_DEFAULT_MEDIA_LINKER", "") == self.name
+
+    def plugin_info_map(self):
+        """Adds extra adapter-specific information to call to the parent fn."""
+
+        result = super(MediaLinker, self).plugin_info_map()
+
+        # something already went wrong, return without doing anything
+        if "ERROR" in result:
+            return
```

Comment:
I think you'll want to `return result` here and not `None`. This is causing issues on line 210 of otiopluginfo.py when iterating the info.